setup.py

```diff
@@ -37,15 +37,15 @@ def read_file(filename):
     # We install psycopg2cffi instead of psycopg2 when dealing with pypy
     # Note: JSONB support landed after psycopg2cffi 2.7.0
     POSTGRESQL_REQUIRES = [
-        'SQLAlchemy',
+        'SQLAlchemy==1.0',  # zopefoundation/zope.sqlalchemy#15
```

Comment:
I am not sure this would take 1.0.13. Also the requirements.txt file seems to say the contrary so if you built the requirements.txt file after this change just keep it like that. I would probably change it with ``<=1.1.0`` since ``1.0.13`` might be the last but we are not sure about it.